tensorflow/contrib/cmake/external/highwayhash.cmake

```diff
@@ -12,9 +12,9 @@ set(highwayhash_STATIC_LIBRARIES
 )
 set(highwayhash_INCLUDES ${highwayhash_BUILD})
 
-set(highwayhash_HEADERS
-    "${highwayhash_BUILD}/highwayhash/*.h"
-)
+file(GLOB highwayhash_HEADERS RELATIVE
+     "${highwayhash_BUILD}/highwayhash"
+     "${highwayhash_BUILD}/highwayhash/*.h")
```

Comment:
I'm not sure why this makes a difference - and it seems to be failing on the CI build :(. Is it a matter of CMake version inconsistency?

I also had to modify this file to get the right thing to happen on Windows: the replacement I used there was:

```cmake
add_custom_command(TARGET highwayhash_copy_headers_to_destination PRE_BUILD
COMMAND ${CMAKE_COMMAND} -E copy_directory ${highwayhash_INSTALL}/include/ ${highwayhash_INCLUDE_DIR}/highwayhash)
```

Would that maybe work in both cases?